src/shared/components/query/CaseSetSelector.tsx

```diff
@@ -68,7 +76,7 @@ export default class CaseSetSelector extends QueryStoreComponent<{}, {}>
 				<ReactSelect
 					value={this.store.selectedSampleListId}
 					options={this.caseSetOptions}
-					clearable={this.store.selectedSampleListId != this.store.defaultSelectedSampleListId}
+					filterOption={(opt: ReactSelectOptionWithName, filter)=>_.includes(opt.textLabel.toLowerCase(), filter)}
```

Comment:
Up to discussion whether we should write a unit test for something like this.   It seems to me that a unit test should: 
1. validate internal logic
2. protect against changes to external code on which this depends.  
For #1 there really isn't any logic here.  For #2 the only external dep here (that is our code) is existnence of textLabel, which is potected by TS.  discuss